assets/tests/components/detours/__snapshots__/detoursListPage.openDetour.test.tsx.snap

```diff
@@ -379,7 +2122,7 @@ exports[`Detours Page: Open a Detour renders detour details modal to match mocke
           >
             Last edited
           </strong>
-          35 hours ago
+          26 hours ago
```

Comment:
huh, why did this change? I don't see any change to the time set in the tests.

Reply:
Ah yes, it's because I switched to using a factory I drew up in a prior PR, and the factory had a different time than the original snapshot for this test